mongofiles/mongofiles.go

```diff
@@ -57,19 +60,40 @@ type MongoFiles struct {
 	// filename in GridFS
 	FileName string
 
-	//ID to put into GridFS
-	Id string
+	// ID to put into GridFS
+	ID string
```

Comment:
process nit: Renaming "Id" to "ID" is fine, but should happened as a minor commit that this PR gets rebased on top of.  It would eliminate of ton of minor diff noise.